bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -330,8 +341,9 @@ public void open() {
             }
 
             ClientUpgradeRequest newRequest = new ClientUpgradeRequest();
+            newRequest.setHeader("user-agent", "openHAB/3.4 com.tibber/1.8.3");
```

Comment:
Same as above, maybe even turn this into a separate method so there's only a single change needed and not multiple values all over the place...

Reply:
Added dynamic bundle version for "user-agent".